code/game/objects/structures/musician.dm

```diff
@@ -50,15 +51,16 @@
 		return
 
 	// now generate name
-	var/soundfile = "sound/[instrumentDir]/[ascii2text(note+64)][acc][oct].[instrumentExt]"
+	var/soundfile = "sound/instruments/[instrumentDir]/[ascii2text(note+64)][acc][oct].[instrumentExt]"
 	soundfile = file(soundfile)
 	// make sure the note exists
 	if(!fexists(soundfile))
 		return
 	// and play
 	var/turf/source = get_turf(instrumentObj)
 	var/sound/music_played = sound(soundfile)
-	for(var/mob/M in hearers(15, source))
+	for(var/A in hearers(15, source))

```

Comment:
This is acceptable because `hearers` only returns a list of mobs; I was shocked the BYOND documentation was accurate.

Also, I tried making this a filtered typecache loop  (typeless to boot) with 28 mobs around; the CPU difference was 0.05, so I couldn't chalk up the difference to being a performance boost or just variation.

Just making it a typeless loop seemed to save about 0.35 CPU, so I figured that *wasn't* just variation, but who knows.